ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py

```diff
@@ -231,6 +231,13 @@ def __print_parsed_manifest_data(self, output: Dict[str, Any]):
         for feature in output['detected-features']:
             lines.append(f'- {feature}')
 
+        dashboards = requirements['grafana-dashboards']
+        if dashboards:
+            lines.append('')
```

Comment:
\n instead is possible?

Reply:
Lines are joined using '\n', so if you'd use '\n' you would get '\n\n' instead. See line 234/241.

Reply:
I mean lines.append('\nDashboards to download:') but up to you :v: